jax/_src/lax/lax.py

```diff
@@ -1060,7 +1060,7 @@ def _get_monoid_reducer(monoid_op: Callable,
   return None
 
 def _get_bitwise_and_identity(dtype: DTypeLike) -> np.ndarray:
-  return np.array(-1, dtype)
+  return np.array(-1).astype(dtype)
 
 def _get_bitwise_or_identity(dtype: DTypeLike) -> np.ndarray:
   return np.array(0, dtype)
```

Comment:
Does this need to be changed as well? Also below, the `np.array(val, dtype)` pattern is used several times.

Reply:
No. It's only because it is now a DeprecationWarning to cast -1 to a uint64/uint32.

Reply:
I see, thanks.